### DISCUSSION TITLE
the eigenvector. out output from the phono function module is a binary file.

### DISCUSSION THREAD

**Discussion by 3183427473** [brucefan1983/GPUMD#694] [Q&A]
In the new version of GPUMD, the eigenvector. out output from the phono function module is a binary file. How can I modify the code to output text format. I looked at this:
https://github.com/brucefan1983/GPUMD/commit/ed48271b7c4a2a1f6ffdf494f08eab8071d15cae#diff -D10e76f254c8da033e299732d5bb89525c7688b67f835c0180e427a2700da73c has now been modified incorrectly.

**Comment by brucefan1983** [comment]
You can follow the diff to modify the code, it should work. You can show the error messges here if you got some.
